Automatic Validation ended with: 
> 2026-03-29T09:50:04.8104376Z ##[error] Manifest Error: The specified installer type does not write to Apps and Features entry. [InstallerType] Value: ***ix Manifest Error: The specified installer type does not write to Apps and Features entry. [InstallerType] Value: ***ix

(Automated response - build 1240.)

--- a/manifests/c/Chill-Astro/Lamina/unp/11.26100.13.0/Chill-Astro.Lamina.unp.installer.yaml
+++ b/manifests/c/Chill-Astro/Lamina/unp/11.26100.13.0/Chill-Astro.Lamina.unp.installer.yaml
@@ -18,10 +18,6 @@ Installers:
   PackageFamilyName: Chill-Astro.Lamina_gb5t6cf7xzm2w
   RestrictedCapabilities:
   - runFullTrust
-  AppsAndFeaturesEntries:
-  - DisplayName: Lamina
-    Publisher: Chill-Astro Software
-    DisplayVersion: 11.26100.13.0
   InstallationMetadata:
     DefaultInstallLocation: '%ProgramFiles%/WindowsApps/Chill-Astro.Lamina_11.26100.13.0_x64__gb5t6cf7xzm2w'
 - Platform:
@@ -37,10 +33,6 @@ Installers:
   PackageFamilyName: Chill-Astro.Lamina_gb5t6cf7xzm2w
   RestrictedCapabilities:
   - runFullTrust
-  AppsAndFeaturesEntries:
-  - DisplayName: Lamina
-    Publisher: Chill-Astro Software
-    DisplayVersion: 11.26100.13.0
   InstallationMetadata:
     DefaultInstallLocation: '%ProgramFiles%/WindowsApps/Chill-Astro.Lamina_11.26100.13.0_arm64__gb5t6cf7xzm2w'
 - InstallerLocale: en-US

